src/nulog-train/README.md

```diff
@@ -0,0 +1,14 @@
+### Build image
+```
+docker build -t nulog-train ./
+```
+
+### Install NVIDIA GPU driver
+```
+kubectl apply -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v0.6.0/nvidia-device-plugin.yml
+```
+
+### Start service
+```
+kubectl apply -f nulog_train.yaml
```

Comment:
this is a `job` not a service. don't want to encourage the user to start this by themselves?